packages/babel-core/src/config/files/module-types.ts

```diff
@@ -68,12 +68,12 @@ function loadCtsDefault(filepath: string) {
       configFile: false,
       sourceType: "script",
       sourceMaps: "inline",
+      sourceFileName: path.basename(filepath),
       presets: [
         [
           getTSPreset(filepath),
           {
-            disallowAmbiguousJSXLike: true,
```

Comment:
Isn't it always disallowed in cts files?

Reply:
conflicts with `allExtensions`, I can put it in `@babel/preset-typescript`.